roles/KubevirtNodeLabeller/tasks/main.yml

```diff
@@ -3,20 +3,47 @@
   k8s:
     state: present
     definition: "{{ item | from_yaml }}"
+    apply: yes
   with_items: "{{ lookup('template', 'kubevirt-node-labeller-roles.yaml.j2').split('\n---\n') | select('search', '(^|\n)[^#]') | list }}"
+  register: roles
 
 - name: Create the node labeller daemon set
   k8s:
     state: present
     definition: "{{ lookup('template', 'kubevirt-node-labeller-ds.yaml.j2') | from_yaml }}"
+    apply: yes
   register: nl
 
+# Actively inject owner references in order to adopt existing resources during an upgrade
+- name: Inject owner references for KubevirtNodeLabellerBundle
+  include_role: 
+    name: ClaimOwnership
+  vars:
+    object: "{{ item }}"
+    owner: "{{ cr_info }}"
+  with_list:
+  - "{{ roles.results[0].result }}" # ServiceAccount
```

Comment:
The issue with roles.results[0].result is that the code will break when someone changes the order of objects in kubevirt-node-labeller-ds.yaml.j2. Why not simply iterate over all of them?

Reply:
The issue is that the file kubevirt-node-labeller-roles.yaml.j2 contains cluster-scoped resources that cannot be owned by a namespaced object

Reply:
Can you filter out those algorithmically?

Reply:
Only way to filter those out is by name/kind